Modules/Core/ImageFunction/include/itkRayCastInterpolateImageFunction.h

```diff
@@ -164,6 +165,13 @@ class ITK_TEMPLATE_EXPORT RayCastInterpolateImageFunction:
     return true;
   }
 
+  SizeType GetRadius() const override
+    {
+    SizeType radius;
+    radius.Fill( NumericTraits< SizeValueType >::max() );
```

Comment:
This looks like it will likely lead to numeric overflow. Could we use the Image size instead?